packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -1,12 +1,15 @@
 import { AmountMath, AmountShape, RatioShape } from '@agoric/ertp';
 import {
+  fromOnly,
+  toOnly,
```

Comment:
I wonder if there's a way to make these more discoverable.

Reply:
They're mentioned in [detailed docs](https://docs.agoric.com/reference/zoe-api/zoe-contract-facet.html#zcf-atomicrearrange-transfers). I think of that as canonical, but everyone else seems to treat [the docs in the .ts file](https://agoric-sdk.pages.dev/types/_agoric_zoe.src_contractFacet_types_ambient.ZCF#__type.atomicRearrange) as definitive.

All is would take is to copy the detailed docs to the .ts file. 

#4291 is relevant.

Reply:
I dunno about "canonical", but the docs in the .ts file are what show up in my IDE while I'm coding. So yes, syncing up a la #4291 should help.